metricbeat/docs/modules/aws.asciidoc

```diff
@@ -19,16 +19,27 @@ module. Please see <<aws-api-requests,AWS API requests>> for more details.
 [float]
 == Module-specific configuration notes
 
+* *AWS Credentials*
+
 The `aws` module requires AWS credentials configuration in order to make AWS API calls.
 Users can either use `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY` and/or
 `AWS_SESSION_TOKEN`, or use shared AWS credentials file.
 Please see <<aws-credentials-config,AWS credentials options>> for more details.
 
+* *regions*
+
 This module also accepts optional configuration `regions` to specify which
 AWS regions to query metrics from. If the `regions` parameter is not set in the
 config file, then by default, the `aws` module will query metrics from all available
 AWS regions.
 
+* *latency*
+
+Some AWS services send monitoring metrics to CloudWatch with a latency to
```

Comment:
Do we already know these AWS services? If so, maybe to list them here to make it easy for new users?

Reply:
So far we have identified one - S3.

Reply:
Yeah right now it's only for S3 metrics. But with `cloudwatch` as the generic metricset, I assume there will be more in the future that has the latency issue.